srcpkgs/pnpm/template

```diff
@@ -10,17 +10,13 @@ maintainer="Bnyro <bnyro@tutanota.com>"
 license="MIT"
 homepage="https://pnpm.io/"
 distfiles="https://registry.npmjs.org/pnpm/-/pnpm-${version}.tgz"
-checksum=5858806c3b292cbec89b5533662168a957358e2bbd86431516d441dc1aface89
+checksum=553e4eb0e2a2c9abcb419b3262bdc7aee8ae3c42e2301a1807d44575786160c9
 python_version=3
 
 do_install() {
 	npm install -g --user root --prefix "${DESTDIR}/usr" "${XBPS_SRCDISTDIR}/${pkgname}-${version}/${pkgname}-${version}.tgz"
 	local _npmdir="${DESTDIR}/usr/lib/node_modules/${pkgname}"
 
-	# Sort and cleanup package.json
-	jq '.|=with_entries(select(.key|test("_.+")|not))' "$_npmdir/package.json" > "$_npmdir/package.json"
-	chmod 644 "$_npmdir/package.json"
-
```

Comment:
Actually, now that I'm reading through the diffs, I'm pretty sure the 8.11.0 issue with the empty package.json was caused precisely by this line, which is now deleted. So that's good, I think.

Reply:
Yes, that's exactly why I removed these lines :+1;